site/docs/getting-started/DesignCards.tsx

```diff
@@ -11,42 +11,45 @@ type CardInfoType = { img: string; altText: string; content?: JSX.Element };
 const cardInfo: CardInfoType[] = [
   {
     img: Assets1,
-    altText: "figma components panel",
+    altText:
+      'Screenshot of Figma\'s assets panel displaying the "Salt: Components Light" library',
```

Comment:
@rufcha I took the liberty of coming up with some alt texts to describe these screenshots for screenreaders (required in order to meet accessibility criteria)

Please let me know if they ~look~ _sound_ OK to you :-)

You can suggest edits directly here in Github by:

1. Hovering over line number to get a + button and clicking it (or clicking and dragiing if you want to cover multiple lines)
2. Clicking the "Add suggestion" button above the comment filed that appears (the leftmost icon that looks like a file with + and - symbols on it)
3. Editng the text between in the "suggestion" block that gets inserted into the comment as desired
4. Submitting your comment